javascript/javascript1/week1/exercises.js

```diff
@@ -0,0 +1,75 @@
+// loginToFreeCodeCamp = sergii-borodin
+
+// Age-ify (A future age calculator)
+const yearOfBirth = 1989;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2027;
+const shouldShowResultInDogYears = true;
+const dogYear = dogYearFuture - dogYearOfBirth;
+if (shouldShowResultInDogYears) {
+  console.log(
+    `Your dog will be ${dogYear * 10} dog years old in ${dogYearFuture}`
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogYear} human years old in ${dogYearFuture}`
+  );
+}
+
+// Housey pricey (A house price estimator)
+const arrOfFriends = ["Peter", "Juliya"];
+const arrOfWidth = [8, 5];
+const arrOfDepth = [10, 11];
+const arrOfHight = [10, 8];
+const arrgOfGardenSizeInM2 = [100, 70];
+const houseCosts = [2500000, 1000000];
+
+for (let i = 0; i < arrOfFriends.length; i++) {
+  const housePriceAccordingFormula =
+    arrOfWidth[i] * arrOfDepth[i] * arrOfHight[i] * 2.5 * 1000 +
+    arrgOfGardenSizeInM2[i] * 300;
+  if (housePriceAccordingFormula > houseCosts[i]) {
+    console.log(arrOfFriends[i] + ", you pay more, then it should be");
+  } else if (housePriceAccordingFormula < houseCosts[i]) {
+    console.log(arrOfFriends[i] + ", you pay less, then it should be");
+  } else {
+    console.log(arrOfFriends[i] + ", you pay as it should be");
+  }
+}
+
+// Ez Namey (Startup name generator)
+const firstWords = [
+  "easy",
+  "awesome",
+  "perfect",
+  "best",
+  "pleasant",
+  "gentle",
+  "smart",
+  "cool",
+  "fun",
+  "fresh",
+];
+const secondWords = [
+  "solution",
+  "corporation",
+  "independence",
+  "future",
+  "communication",
+  "deal",
+  "pleasure",
+  "ambition",
+  "foundation",
+  "decision",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
+const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
```

Comment:
In this case `randomNumber` will be the same for both arrays, so words will always be paired together. You could make it more random by generating 2 random numbers. If you do that, there will also be no reason to declare the `randomNumber` variable, you could just inline the random number generator, like:
```suggestion
const startupName = firstWords[Math.floor(Math.random() * firstWords.length)] + " " + secondWords[Math.floor(Math.random() * secondWords.length)];
```

Note I've also replaced the hard-coded `10` with a call to `.length`, so the arrays can be any length.